File: Programar em C#/07-MaisSobreAsStrings.cs
// YouTube: https://youtu.be/pKGz-CCpye8
// Publicação: https://caffeinealgorithm.com/blog/20211111/mais-sobre-as-strings-em-csharp/

using System;

namespace Base {
  class MaisSobreAsStrings {
    public void Run() {
      string primeiroNome = "Nelson";
      string ultimoNome = "Silva";
      int idade = 25;

      Console.WriteLine($"Nome: {primeiroNome} {ultimoNome}\nIdade: {idade}");

      /*
        Nome: Nelson Silva
        Idade: 25
      */
    }
  }
}

File: Programar em Java/07-MaisSobreAsStrings.java
// YouTube: https://youtu.be/Vk0V-iw_LWo
// Publicação: https://caffeinealgorithm.com/blog/20220214/mais-sobre-as-strings-em-java/

package com.caffeinealgorithm.programaremjava;

public class MaisSobreAsStrings {
  public void Run() {
    String primeiroNome = "Nelson";
    String ultimoNome = "Silva";
    int idade = 25;

    System.out.println("Nome: " + primeiroNome + ' ' + ultimoNome + "\nIdade: " + idade);
    System.out.println(new StringBuilder().append("Nome: ").append(primeiroNome).append(' ').append(ultimoNome).append("\nIdade: ").append(idade));
    System.out.printf("Nome: %s %s\nIdade: %d", primeiroNome, ultimoNome, idade);
  }
}

/*
  Nome: Nelson Silva
  Idade: 25
  Nome: Nelson Silva
  Idade: 25
  Nome: Nelson Silva
  Idade: 25
*/
